### PR TITLE
Update network-exchange-connectivity-matrix-configuration-guide.md

### PR DESCRIPTION
**Pull request by marcopao**
Added DE3 as available endpoint for FR6

### CODE REVIEW

File: Network/network-exchange-connectivity-matrix-configuration-guide.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ New Jersey, NJ|NJ2|NJ2<br>NJ2X|NY1
 Seattle, WA|SE2|SE2|WA1
 London, UK|LO1|LO1|GB3
 London, UK|LO6|LO6|GB1
-Frankfurt, DE|FR6|FR6|DE1
+Frankfurt, DE|FR6|FR6|DE1|DE3
```
Comment (chrislittle) [inline Network/network-exchange-connectivity-matrix-configuration-guide.md]: this should be DE1<br>DE3 right? you want to list both sides in the same table box. its not rendering properly as you submitted

Comment (chrislittle) [reply Network/network-exchange-connectivity-matrix-configuration-guide.md]:
```
DE1<br>DE3
```
sorry

**Comment by marcopao** [comment]
Chris,


Missed your email yesterday.   Yes, would like to Have DE1 and DE3 listed together, DE1<br>DE3 - sorry I messed it up.

Will fix now.

Thanks,


Marco

________________________________

**Comment by chrislittle** [comment]
closing PR as it was replaced with https://github.com/CenturyLinkCloud/PublicKB/pull/2220

**Pull request closed without merging**